src/lib/server/phoenixd.ts

```diff
@@ -34,6 +34,18 @@ export async function phoenixdBalance(): Promise<{ balanceSat: number; feeCredit
 	return await res.json();
 }
 
+export async function phoenixdLndAddress(): Promise<string> {
+	const res = await fetch(`${runtimeConfig.phoenixd.url}/getlnaddress`, {
+		headers: {
+			Authorization: `Basic ${Buffer.from(`:${runtimeConfig.phoenixd.password}`).toString(
+				'base64'
+			)}`
+		}
+	});
+
+	return await res.text();
+}
```

Comment:
Does phoenixd on a fresh install (no channel created because no payment received) return a valid response?

if not, what does an error response looks like?

This should throw on errors

Reply:
It returns a message saying there is no channel.